series/series.go

```diff
@@ -647,3 +651,64 @@ type indexedElements []indexedElement
 func (e indexedElements) Len() int           { return len(e) }
 func (e indexedElements) Less(i, j int) bool { return e[i].element.Less(e[j].element) }
 func (e indexedElements) Swap(i, j int)      { e[i], e[j] = e[j], e[i] }
+
+// StdDev calculates the standard deviation of a series
+func (s Series) StdDev() float64 {
+	stdDev := stat.StdDev(s.Float(), nil)
+	return stdDev
+}
+
+// Mean calculates the average value of a series
+func (s Series) Mean() float64 {
+	stdDev := stat.Mean(s.Float(), nil)
+	return stdDev
+}
+
+// Max return the biggest element in the series
+func (s Series) Max() Element {
+	if s.elements.Len() == 0 {
+		return nil
+	}
+	max := s.elements.Elem(0)
+	for i := 0; i < s.elements.Len(); i++ {
+		elem := s.elements.Elem(i)
+		if elem.Greater(max) {
+			max = elem
+		}
+	}
+	return max
+}
+
+// Min return the lowest element in the series
+func (s Series) Min() Element {
+	if s.elements.Len() == 0 {
+		return nil
+	}
+	min := s.elements.Elem(0)
+	for i := 0; i < s.elements.Len(); i++ {
+		elem := s.elements.Elem(i)
+		if elem.Less(min) {
+			min = elem
+		}
+	}
+	return min
+}
+
+// Quantile returns the sample of x such that x is greater than or
+// equal to the fraction p of samples.
+// Note: gonum/stat panics when called with strings
+func (s Series) Quantile(p float64) float64 {
+	if s.Type() == String {
+		math.NaN()
+	}
+
+	original := s.Float()
```

Comment:
To order a Series you can use the existing methods:

`ordered := s.Subset(s.Order(false)).Float()`